WebApplication/Job/AdoptProjectWithParametersJobItem.cs

```diff
@@ -53,11 +53,11 @@ public override async Task ProcessJobAsync(IResultSender resultSender)
 
                 Logger.LogInformation($"ProcessJob (AdoptProjectWithParameters) {Id} for project {payload.Name} started.");
 
-                ProjectStorage projectStorage = await _projectService.AdoptProjectWithParametersAsync(payload);
+                var projectDto = await _projectService.AdoptProjectWithParametersAsync(payload, _dtoGenerator);
 
                 Logger.LogInformation($"ProcessJob (AdoptProjectWithParameters) {Id} for project {payload.Name} completed.");
                 
-                await resultSender.SendSuccessAsync(Tuple.Create(_dtoGenerator.ToDTO(projectStorage), payload));
+                await resultSender.SendSuccessAsync(Tuple.Create(projectDto, payload));
```

Comment:
Payload doesn't contain the right set of parameters.

Reply:
It should be fine now, all credit to Miro